src/ThreeEditor/Simulation/Base/SimulationPoints.ts

```diff
@@ -150,6 +150,11 @@ export abstract class SimulationPoints
 			this.overrideHandler
 		) as this;
 	}
+
+	// eslint-disable-next-line class-methods-use-this
+	duplicate(): SimulationPoints {
+		throw new Error('Method not implemented.');
+	}
```

Comment:
It's a little weird to have a `Method not implemented` error in an abstract class. That's what abstract methods are for.
```suggestion
	abstract duplicate(): SimulationPoints;
```